components/grid/export/pdf.md

```diff
@@ -10,17 +10,223 @@ position: 1
 
 # PDF Export
 
-The [Telerik Document Processing tools](slug://dpl-in-blazor) that come with your Blazor license let you generate a PDF file based on the data in the grid.
+You can export the Grid to PDF with a click of a button. The current filter, sort, page, grouping, column order and column size are applied to the exported PDF document.
```

Comment:
`a click` indicates a random, unidentified click, `the` is more definitive. Note that the [Style Guide](https://docs.telerik.com/style-guide/punctuation#series) calls for the use of the serial comma (which is counter-intuitive for some other languages).

```suggestion
You can export the Grid to PDF with the click of a button. The current filter, sort, page, grouping, column order, and column size are applied to the exported PDF document.
```